CONTRIBUTING.md

```diff
@@ -18,7 +18,8 @@ To get started quickly:
 1. Navigate to [https://github.com/deephaven/web-client-ui](https://github.com/deephaven/web-client-ui).
 2. Click `Fork` in the top right corner.
 3. `git clone git@github.com:<username>/web-client-ui.git`
-4. Commit changes to your own branches in your forked repository.
+4. In order to view the UI, you must also be running the [deephaven-core](https://github.com/deephaven/deephaven-core). deephaven-core provides REST APIs that the web-client-ui depends upon. An easy way to get started is to follow the steps for [Launch Python](https://github.com/deephaven/deephaven-core#launch-python).
```

Comment:
```suggestion
4. In order to use the UI, you must also be running a [deephaven-core](https://github.com/deephaven/deephaven-core) server on port 10000. The server provides APIs that the web-client-ui depends upon. An easy way to get started is to follow the steps for [Launch Python](https://github.com/deephaven/deephaven-core#launch-python).
```

Technically it's not a rest api, it's a gRPC api over web socket. Just removing that detail and describing it generically is fine. Thanks for this.

Reply:
Thank you.  I like your edits